bika/lims/content/bikasetup.py

```diff
@@ -541,6 +541,17 @@ class PrefixesField(RecordsField):
         )
     ),
     StringField(
+        'AutoImportInterval',
+        schemata="Analyses",
+        default="30",
+        widget=StringWidget(
+            label=_("Interval of Auto-Importing Files in minutes"),
+            description=_("System will upload result files of different \
+                          instruments/interfaces periodically in the interval \
+                          of this value (Minutes).")
```

Comment:
Which is the value should I set if I want to disable the AutoImportInterval? Need to add this in the descrioption. Also, this should be the default value instead of 30.